release/gff/gff.har.gff_harari/source/welcome.htm

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>Start Using GFF Amharic</title>
+  <style type="text/css">
+    p { font: 10pt Tahoma; }
+    h1 { font: bold 16pt Tahoma; color: #4444cc; margin-bottom: 2px }
+    h2 { font: bold 12pt Tahoma; color: #4444cc; }
+  </style>
+</head>
+<body>
+
+<h1>Start Using GFF Harari</h1>
+
+<p>
+    This lexical model is intended for use with the Harari Keyman Keyboard on mobile devices, see:
+    <a href="https://keyman.com/harari/">https://keyman.com/harari/</a>

```

Comment:
I don't know if this is the intended keyboard
https://keyman.com/keyboards/gff_harege_fidela

Reply:
The file also starts with a BOM (FEFF) but I think that shouldn't matter.